### DISCUSSION TITLE
Andere Seite

### DISCUSSION THREAD

**Discussion by kironia** [eltikia/eltikia.github.io#7] [Q&A] [answered]
Gibt es mehr zu sehen?

**Comment by Stagyrite** [comment, marked as answer]
Ja, natürlich. Ich habe eine andere Seite [Kironia.GitHub.io](https://kironia.github.io/ "Nostalgie-Ecke von Stagirit") gebaut. Es ist nur eine Kopie. Es ist doch [Eltikia.GitHub.io](https://eltikia.github.io/ "Nostalgie-Ecke von Stagirit"), aber mit ein bisschen Hilfe von @kironia.
Man kann diese Nostalgie-Ecke an der [YouTube](https://www.youtube.com/@eltikia "Nostalgie-Ecke von Stagirit - YouTube") schauen. Dort finden Sie ein bisschen Kommentare.
Diese Internetseite hat seine Gesichte. Ich habe viele altmodische Computerspiele gespielt. Es ist nicht ganz und gar über Nostalgie, denn der Inhalt macht mein GitHub aktiv.
